tests/dynamic_settings/test_dynamic_settings.py

```diff
@@ -148,3 +148,11 @@ def test_should_auto_casting_values_in_dynamic_storage(
         assert reader.get(key) == expected_setting
 
         assert reader._dict['COMPLEX'] != expected_setting
+
+    def test_should_get_none_when_auto_casting_for_unknown_key_dynamic_storage(
+        self, settings_dict, reader
+    ):
+        key = 'NON-EXISTING-KEY'
+        reader.auto_casting = True
+
+        assert reader.get(key) == None
```

Comment:
is better to test `None`, `True` and `False` with `is` instead of `==`:
```diff
- assert reader.get(key) == None
+ assert reader.get(key) is None
```